docs/index.rst

```diff
@@ -4,68 +4,130 @@
 Zarr
 ====
 
-Zarr is a format for the storage of chunked, compressed, N-dimensional arrays.
-These documents describe the Zarr format and its Python implementation.
+Zarr is a format for the storage of chunked, compressed, N-dimensional arrays
+inspired by `HDF5 <https://www.hdfgroup.org/HDF5/>`_, `h5py
+<https://www.h5py.org/>`_ and `bcolz <https://bcolz.readthedocs.io/>`_.
+
+Development of Zarr is supported by the
+`MRC Centre for Genomics and Global Health <https://www.cggh.org>`_
+and the `Chan Zuckerberg Initiative <https://chanzuckerberg.com/>`_.
```

Comment:
Should we add NumFOCUS here?

Reply:
Added. (a) We should review at some point what's said here and on https://zarr.dev and (b) we should think about collecting logos for those that support the project. :smile: